src/lazy/binary/raw/v1_1/sequence.rs

```diff
@@ -27,6 +29,12 @@ impl<'top> LazyContainerPrivate<'top, BinaryEncoding_1_1> for LazyRawBinaryList_
     }
 }
 
+impl<'top> LazyRawContainer<'top, BinaryEncoding_1_1> for LazyRawBinaryList_1_1<'top> {
+    fn as_value(&self) -> <BinaryEncoding_1_1 as LazyDecoder>::Value<'top> {
+        self.sequence.value
+    }
+}
+
```

Comment:
🗺️ All lazy container types can now be converted back to the original (less specific) 'value' representation.